parts/kubernetesagentcustomdata.yml

```diff
@@ -141,7 +141,11 @@ write_files:
 
 runcmd:
 - apt-get update
-- apt-get install -y apt-transport-https ca-certificates
+- apt-get install -y apt-transport-https ca-certificates nfs-common
```

Comment:
For now I was leaving it off. I can't think of a use case, and I was going for the principle of smallest necessary change.